### PR TITLE
fix declaration of vars

### PR DESCRIPTION
**Pull request by Netzpfuscher**
Fix messed up interface

### CODE REVIEW

File: MESC_Common/Src/MESCsin_lut.c [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ ******************************************************************************
+
+ * MESCsin_lut.c
+ *
+ *  Created on: 25 Jul 2020
+ *      Author: David Molony
```
Comment (cod3b453) [inline MESC_Common/Src/MESCsin_lut.c:22]: This description is wrong and should be updated accordingly

File: MESC_Common/Src/MESCsin_lut.c [hunk 2]
```diff
@@ -346,3 +352,92 @@ float sinwave[321] = {0,
                       0.9987954562,
                       0.9996988187,
                       1};
+void sin_cos_fast( int16_t angle , float * sin, float * cos)
+{
+	*sin = sinwave[angle >> 8];
+	*cos = sinwave[(angle >> 8) + 64];
```
Comment (cod3b453) [inline MESC_Common/Src/MESCsin_lut.c:358]: Strictly speaking, either `angle` should be `uint16_t` or `>> 8 `should be `/ 256`

File: MESC_Common/Src/MESCsin_lut.c (hunk 1 above)
Comment (Netzpfuscher) [reply MESC_Common/Src/MESCsin_lut.c:22]: see new commit

File: MESC_Common/Src/MESCsin_lut.c (hunk 2 above)
Comment (Netzpfuscher) [reply MESC_Common/Src/MESCsin_lut.c:79]: see new commit

**Pull request merged**